core-bundle/src/Resources/contao/library/Contao/Image.php

```diff
@@ -456,6 +456,7 @@ public function executeResize()
 					->setImagineOptions(System::getContainer()->getParameter('contao.image.imagine_options'))
 					->setTargetPath($this->targetPath ? $this->strRootDir . '/' . $this->targetPath : null)
 					->setBypassCache(System::getContainer()->getParameter('contao.image.bypass_cache'))
+					->setSkipIfDimensionsMatch(true)
```

Comment:
Why?

Reply:
The deprecated legacy image class should work as it did in older versions. Also because it is not possible to control this flag with the legacy class.

Reply:
Agreed. 👍